indigo/distribution.yaml

```diff
@@ -3533,6 +3533,18 @@ repositories:
       url: https://github.com/ual-arm-ros-pkg/mvsim.git
       version: master
     status: maintained
+  myo_ros:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/clearpath-gbp/myo_ros-release.git
+      version: 0.1.2-0
+    source:
+      type: git
+      url: https://github.com/clearpathrobotics/myo_ros.git
+      version: master
+    status: maintained
+    status_description: New release. Adding examples planned, but low-priority.
```

Comment:
@rgariepy Thanks for updating the pull request. Just to follow up on @dirk-thomas's request, this is what he is talking about:

From http://www.ros.org/reps/rep-0137.html#release-file
> status_description: an optional field describing in a short sentence the current status of the repository. For example detailing the reason for EOL and the recommended upgrade path.

Though it doesn't restrict using this field for anything, I believe the intention was that this should only really be used when detailing why something is EOL and what users should do instead.